docs/src/docs/asciidoc/getting-help.adoc

```diff
@@ -3,4 +3,21 @@
 :toc: left
 :toclevels: 1
 
-This page is under construction.
+[[getting-help-community]]
+== Community
+
+Welcome to the https://docs.spring.io/spring-security/reference/community.html[Spring Security Community]!
```

Comment:
Change the exclamation point to a period. Technical documentation has a more formal expectation than an exclamation point.